babel.config.js

```diff
@@ -0,0 +1,11 @@
+const presets = [
+  [
+    "@babel/env",
+    {
+      useBuiltIns: "usage",
+      corejs: 3,
+    },
+  ],
+];
+
+module.exports = { presets };
```

Comment:
This looks great to me. But I think a .babelrc file would have sufficed to avoid the export. Great job

Reply:
Hello, Thank you for your feedback. Babel recommends babel.config.js for configuration. You can check this configuration page https://babeljs.io/docs/en/configuration